src/services/tripService.js

```diff
@@ -27,8 +27,7 @@ export const findTripById = async (tripId) => {
           model: Accomodation,
           attributes: ['id', 'name']
         }
-      ],
-      attributes: { exclude: ['accomodationId'] }
```

Comment:
No need to remove this line as the id of accommodation is included in the join statement above on line 28